content/influxdb/cloud/reference/internals/security.md

```diff
@@ -0,0 +1,240 @@
+
+InfluxDB Cloud is available on the following cloud providers:
+- Amazon Web Services (AWS)
+- Microsoft Azure
+- Google Cloud Platform (GCP)
+
+To ensure data security, availability, and durability:
+- Each instance is isolated and protected in its own virtual private cloud (VPC)
+- Clusters are deployed across multiple provider availability zones
+
+### Amazon Web Services (AWS)
+
+An instance of InfluxDB Cloud consists of microservices in Kubernetes.
+Each VPC within AWS is segmented into public and private subnets:
+
+- The public subnet contains resources exposed to the public internet, including load balancers and network address translation (NAT) gateways.
+- The private subnet contains supporting infrastructure (e.g., compute, storage) not exposed to the public internet.
+
+To ensure fault tolerance across the data layer, clusters are deployed across multiple availability zones.
+AWS data centers are compliant with many physical and information security standards.
+For detail about AWS's physical security and data center protocols, see [AWS's Compliance](https://aws.amazon.com/compliance/).
+
+### Google Cloud Platform (GCP)
+
+In Google Cloud Platform (GCP), InfluxDB Cloud uses the Google Kubernetes Engine (GKE)
+and Google Compute Engine to deploy individual cluster components.
+Clusters are isolated at the project level
+to enhance access controls, data governance, and support auditing.
+To ensure fault tolerance across the data layer, clusters are deployed across multiple availability zones.
+
+Google Cloud Platform data centers are compliant with many physical and information security standards.
+For detail about physical security in GCP data centers, see [Google's Compliance website](https://cloud.google.com/security/compliance/).
+
+### Microsoft Azure
+
+In Microsoft Azure, InfluxDB Cloud uses Azure Kubernetes Service (AKS)
+and Azure Virtual Machines to deploy individual cluster components.
+To support auditing and authorization control within Azure,
+clusters are deployed into dedicated VNets within each region.
+To ensure fault tolerance across the data layer, clusters are deployed across multiple availability zones.
+
+Microsoft Azure data centers are compliant with many physical and information security standards.
+For detail about physical security within Microsoft Azure data centers, see [Microsoft's Compliance website](https://www.microsoft.com/en-us/trust-center/compliance/compliance-overview).
+
+### Data encryption
+
+InfluxDB Cloud enforces TLS encryption for data in transit from all clients, including Telegraf agents, browsers, and custom applications.
+Requests using TLS 1.2 or earlier are rejected.
+
+By default, data at rest is encrypted using strong encrypted methods (AES-256) within AWS, GCP, and Microsoft Azure.
+
+User managed encryption keys are not supported in InfluxDB Cloud at this time.
+
+## Application and service security
+
```

Comment:
we may want to add an intro sentence (so we don't have double headers) --something like We observe the following application and service security measures:
- Internal access controls
- ....